core/src/main/java/org/apache/iceberg/ManifestFilterManager.java

```diff
@@ -77,9 +78,12 @@ public String partition() {
   private boolean failMissingDeletePaths = false;
   private int duplicateDeleteCount = 0;
   private boolean caseSensitive = true;
+  private boolean allDeletesReferenceManifests = true;
+  private boolean useReferencedManifests = false;
```

Comment:
I wonder if calling this var `trustXXX` would make it easier to understand. Essentially, this indicates whether we can trust the set of referenced manifests.

Reply:
Agree that's a better name, `use` doesn't really indicate much come to think of it. A natural question is "how does it get used", and trust implies a source of truth